modules/extensions/src/data-filter/aggregator.ts

```diff
@@ -19,7 +18,8 @@ const float component = 1.0 / 255.0;
 
 void main() {
   #ifdef FLOAT_TARGET
-    dataFilter_value *= float(filterIndices != filterPrevIndices);
+    // BUG: always 0 for some reason
+    // dataFilter_value *= float(filterIndices != filterPrevIndices);
```

Comment:
@Pessimistress any idea why this is broken? I've tried different variants of the check and it always evaluates to 0. Perhaps there is something wrong with using `vertexOffset`?

Reply:
Subtle catch of v9 - when an attribute's descriptor contains `shaderAttributes`, the sub-attributes cannot use the same name.

Reply:
That is subtle indeed :) would be nice if the types could catch this